### DISCUSSION TITLE
WireGuard does not seem to be running

### DISCUSSION THREAD

**Discussion by indianmaksi** [angristan/wireguard-install#402] [Help]
Kali machine terminal does not specify
Welcome to WireGuard-install!
The git repository is available at: https://github.com/angristan/wireguard-install

It looks like WireGuard is already installed.

What do you want to do?
   1) Add a new user
   2) Revoke existing user
   3) Uninstall WireGuard
   4) Exit
Select an option [1-4]: 1

Tell me a name for the client.
The name must consist of alphanumeric character. It may also include an underscore or a dash and can't exceed 15 chars.
Client name: tor
Client's WireGuard IPv4: 10.66.66.2
Client's WireGuard IPv6: fd42:42:42::2
./wireguard-install.sh: line 282: wg: command not found
./wireguard-install.sh: line 283: wg: command not found
./wireguard-install.sh: line 284: wg: command not found
./wireguard-install.sh: line 322: wg: command not found
./wireguard-install.sh: line 322: wg-quick: command not found

Here is your client config file as a QR Code:
./wireguard-install.sh: line 326: qrencode: command not found
It is also available in /home/mak/wg0-client-tor.conf

![image](https://user-images.githubusercontent.com/37648323/144791352-d006add4-d256-4db5-9ba2-80a8d0ba7e54.png)

**Comment by gilboz** [comment]
Seems like you don't have the relevant packages installed, i don't know how you've gotten to this situation.. check if `/etc/wireguard/params` file exists. if so delete it and re-run the script.
